doc/source/whatsnew/v0.23.2.txt

```diff
@@ -25,8 +25,8 @@ Fixed Regressions
 Performance Improvements
 ~~~~~~~~~~~~~~~~~~~~~~~~
 
-- Improved performance of membership checks in :class:`CategoricalIndex`
-  (i.e. ``x in ci``-style checks are much faster). :meth:`CategoricalIndex.contains`
+- Improved performance of membership checks in :class:`CategoricalIndex` and :class:`Categorical`
+  (i.e. ``x in cat``-style checks are much faster). :meth:`CategoricalIndex.contains`
```

Comment:
This is just a cleanup of the whatsnew for #21508, not related to this PR.

Reply:
put this in a separate PR then.